src/lib/components/parameters.svelte

```diff
@@ -141,6 +149,30 @@
     );
     exitConfigurationMode();
   };
+
+  const bluetoothOn = () => {
```

Comment:
Similar to the above, could these be verbs?

```suggestion
  const enableBluetooth = () => {
```